tools/scripts/build-website.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -ex
+
+VERSION_NUMBER=$1
+if [[ "$VERSION_NUMBER" == "" ]]; then
+  VERSION_NUMBER=master
+fi
+
+BASE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+echo "Updating online runner ..."
+python3 "$BASE_DIR/add_online_runner.py"
+
+# Configure Git User, mike will create gh-pages branch
+git config user.name "nobody"
+git config user.email "nobody@localhost"
+
+echo "Building docs for $VERSION_NUMBER"
+mike deploy "$VERSION_NUMBER" -F docs/mkdocs.yml -b gh-pages
+git checkout gh-pages
+
+echo "Attempting to update versions.json"
+aws s3 cp s3://djl-ai/documentation/nightly/versions.json . 
```

Comment:
1. Can we just generate this `versions.json` file without download it?
2. We don't need use awscli to download it, we can use curl:
```
curl -O https://docs.djl.ai/versions.json
```

Reply:
Probably, I'll look into using curl, I am unfamiliar with it